ProjectTaskTracker.js

```diff
@@ -1,77 +1,152 @@
+const toDoList = document.getElementsByClassName("toDoList")[0];
+const inProgressList = document.getElementsByClassName('inProgressList')[0];
+const doneList = document.getElementsByClassName('doneList')[0];
+
+let toDoTaskNumber = 0;
+let projectNumber = 0;
 
 function addToDoTextarea() {
-	const newTask = document.createElement('textarea');
-	newTask.classList.add("createNewTask" + toDoTaskNumber);
+ 	const newTask = document.createElement('textarea');
+	newTask.classList.add("createNewTask" + toDoTaskNumber, "tasks");
 	return newTask;
 }
 
-function addToDoTask(){
-	const newTaskForm = document.createElement('textarea');
-	newTaskForm.classList.add("createNewTask" + toDoTaskNumber);
-	newTaskForm.classList.add("tasks");
-	newTaskForm.setAttribute("draggable", "true");
+function addNewProjectTextarea() {
+	const newProjectTextarea = document.createElement('textarea');
+	newProjectTextarea.classList.add("createNewProject" + projectNumber, "projects");
+	return newProjectTextarea
+}
 
-	
-	const toDoList = document.getElementsByClassName("toDoList")[0];
+function addToDoTask(){
+	const newTaskForm = addToDoTextarea();
 	toDoList.appendChild(newTaskForm);
 	toDoTaskNumber++;
 }
 
 function addNewProject() {
-	const newProject = document.createElement('textarea');
-	newProject.classList.add("createNewProject" + projectNumber);
-	newProject.classList.add("projects");
-	newProject.setAttribute("draggable", "true");
-
+	const newProject = addNewProjectTextarea();
 	const projectList = document.getElementsByClassName("sideBar")[0];
 	projectList.appendChild(newProject);
 	projectNumber++;
 }
 
 addTaskButton.addEventListener('click', ()=> addToDoTask());
 addNewProjectButton.addEventListener('click', ()=> addNewProject());
+/*---------------------------------------------------------------------------------------*/
+
+/*----------------------------develop draggable event------------------------------------*/
+const toDoTaskDraggable = new Draggable.Draggable(toDoList, {
+  		draggable: '.tasks'
+	});
+
+const progressTaskDraggable = new Draggable.Draggable(inProgressList, {
+  		draggable: '.tasks'
+	});
 
-// //make Project elements draggable
-// const projectElement = document.getElementsByClassName('sideBar')[0];
-// const projectDraggable =new Draggable.Draggable(projectElement, {
-// 	draggable: '.projects'
-// 	});
+const doneTaskDraggable = new Draggable.Draggable(doneList, {
+  		draggable: '.tasks'
+	});
 
-// //make tasks draggable
-// const taskElement = document.getElementsByClassName('toDoList')[0];
-// const tasksDraggable = new Draggable.Draggable(taskElement, {
-// 	draggable: '.tasks'
-// 	});
+const threeLists = [toDoList, inProgressList, doneList];
 
-const tasks = document.getElementsByClassName('tasks')[0];
-if(tasks){
-	tasks.on('drag:start', ()=> console.log('dragging'));
+let currentMousePositionX = 0;
+let currentMousePositionY = 0;
 
+let currentDraggingItem;
+let dragFromList;
+
+function getDraggingElement(event) {
```

Comment:
This isn't really "getting" anything. This is finding and assigning. You can either have this return the element or rename this